README.md

```diff
@@ -0,0 +1,165 @@
+[![Build Status](https://travis-ci.com/oasis-roles/ocp_pull_secrets.svg?branch=master)](https://travis-ci.com/oasis-roles/ocp_pull_secrets)
+
+ocp_pull_secrets
+================
+
+Get pull secrets to be used by OpenShift Container Platform (OCP) 4.x
+to pull images from registries that require authentication. This is done with
+your offline access token found at https://cloud.redhat.com/openshift/token
+(login required), and according to the documentation found in the OCP
+[install guides](https://cloud.redhat.com/openshift/install).
+
+**Note**: As this role demonstrates, the offline access token can be used to
+grant anyone the ability to authenticate to APIs, such as `api.openshift.com`,
+as the owner of the offline access token. **Protect this token as you would
```

Comment:
It might be cool to mention and/or link to Ansible Vault here just as a handy reference.

https://docs.ansible.com/ansible/latest/user_guide/vault.html